### DISCUSSION TITLE
Patrol CLI not updating

### DISCUSSION THREAD

**Discussion by akagupta9** [leancodepl/patrol#1099] [Q&A] [answered]
✓ New patrol_cli version is available (1.0.10) (0.5s)
✓ Updated patrol_cli to version 1.0.10 (3.2s)
✗ Failed to download artifacts for version 1.0.10 (0.6s)
Exception: Failed to download Artifact.archive(name: ios, version: 1.0.10, ext: null) from https://github.com/leancodepl/patrol/releases/download/patrol_cli-v1.0.10/ios-1.0.10.zip
Exception: Failed to download Artifact.archive(name: ios, version: 1.0.10, ext: null) from https://github.com/leancodepl/patrol/releases/download/patrol_cli-v1.0.10/ios-1.0.10.zip
#0      ArtifactsRepository._downloadArtifact (package:patrol_cli/src/common/artifacts_repository.dart:295:7)
<asynchronous suspension>
#1      Future.wait.<anonymous closure> (dart:async/future.dart:521:21)
<asynchronous suspension>
#2      ArtifactsRepository.downloadArtifacts (package:patrol_cli/src/common/artifacts_repository.dart:279:5)
<asynchronous suspension>
#3      UpdateCommand.run (package:patrol_cli/src/features/update/update_command.dart:76:7)
<asynchronous suspension>
#4      CommandRunner.runCommand (package:args/command_runner.dart:209:13)
<asynchronous suspension>
#5      PatrolCommandRunner.runCommand (package:patrol_cli/src/command_runner.dart:291:18)
<asynchronous suspension>
#6      PatrolCommandRunner.run (package:patrol_cli/src/command_runner.dart:235:18)
<asynchronous suspension>
#7      patrolCommandRunner (package:patrol_cli/src/command_runner.dart:52:14)
<asynchronous suspension>
#8      main (file:///Users/akankshagupta/.pub-cache/hosted/pub.dartlang.org/patrol_cli-0.8.2+1/bin/main.dart:6:20)
<asynchronous suspension>

**Comment by bartekpacia** [comment]
You're migrating from an old version.

Please reinstall:

```console
dart pub global deactivate patrol_cli
```

```console
dart pub global activate patrol_cli
```

**Comment by akagupta9** [comment]
I tried same but than it's not installing latest version.

**Comment by akagupta9** [comment]
https://github.com/leancodepl/patrol/releases/download/patrol_cli-v1.0.10/ios-1.0.10.zip

This link doesn't exist at all

**Comment by bartekpacia** [comment, marked as answer]
In that case, you're probably using an old Flutter version. As you can see in [the README](https://pub.dev/packages/patrol_cli), `patrol_cli` requires at least Dart 2.18, which is bundled with Flutter 3.3.

**Comment by akankshajify** [comment]
Thanks, It worked.
